email_helper.user.js

```diff
@@ -0,0 +1,123 @@
+// ==UserScript==
+// @name         Tau Station Email Helper
+// @namespace    http://tampermonkey.net/
+// @downloadURL  https://github.com/taustation-fan/userscripts/raw/master/email_helper.user.js
+// @version      0.1
+// @description  Provides multi-email forwarding functionality to Tau Station email system
+// @author       Sergey Kudriavtsev <https://github.com/dot-sent>
+// @match        https://alpha.taustation.space/email/*
+// @grant        none
+// @require      https://code.jquery.com/jquery-3.3.1.min.js
+// @require      https://unpkg.com/turndown/dist/turndown.js
+// ==/UserScript==
+
+var localStorageKey = 'emailsToForward';
+
+var turndownService = new TurndownService();
+
+function extractEmailText(emailId) {
+    var emailBody = $('#email-body-' + emailId);
+    var emailHeader = $('#email-' + emailId);
+    if (emailBody.length == 0) { // This is the MAIN/currently opened email in conversation
+        emailBody = $(emailHeader).next();
+    }
+    var emailComponents = [];
+    var emailBodyAuthorComponents = emailBody.children('.comment-author');
+    emailComponents.push(emailHeader.children('.comment-date')[0].innerHTML);
```

Comment:
If you're using jquery, you can save some typing by using `.html()` instead of `.innerHTML`.